### PR TITLE
Updated post Chef-18 to clarify how to release

### PR DESCRIPTION
**Pull request by johnmccrae**
Signed-off-by: John <john.mccrae@progress.com>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: docs/dev/how_to/releasing_chef_infra.md [hunk 1]
```diff
@@ -29,10 +31,18 @@ If there are any new or updated resources, the docs site will need to be updated
 #### Resource Documentation Automation
 
 1. Run `rake docs_site:resources` to generate content to a `docs_site` directory
+   1. WARNING: Any resource that inherits it's basic structure from a parent resource is likely to comingle settings from both when you run that command. For example, there are 17-20 resources that consume the basic package.rb resource file. As a consequence, ALL of the children will most likely have pulled in properties and actions that do not apply to them. You have to be careful here.
```
Comment (tpowell-progress) [inline docs/dev/how_to/releasing_chef_infra.md:34]: should be `commingle`

File: docs/dev/how_to/releasing_chef_infra.md [hunk 2]
```diff
@@ -29,10 +31,18 @@ If there are any new or updated resources, the docs site will need to be updated
...
+
 2. Compare the relevant generated files to the content in the `content/resources` directory within the [chef-web-docs repo](https://github.com/chef/chef-web-docs/). The generated files are missing some content, such as action descriptions, and don't have perfect formatting, so this is a bit of an art form.
+   1. This will take time - expect to use 2-4 days of going through the new docs to ensure you aren't accidentally overwriting things or ignoring important updates
+   2. One tool you can use to help yourself is Beyond Compare. If you have to buy it, it's like 20$
```
Comment (tpowell-progress) [inline docs/dev/how_to/releasing_chef_infra.md:38]: `20$` -> `$20`

**Review by tpowell-progress — CHANGES_REQUESTED** [review]
See suggestions

File: docs/dev/how_to/releasing_chef_infra.md (hunk 1 above)
Comment (johnmccrae) [reply docs/dev/how_to/releasing_chef_infra.md]: corrected

File: docs/dev/how_to/releasing_chef_infra.md (hunk 2 above)
Comment (johnmccrae) [reply docs/dev/how_to/releasing_chef_infra.md:38]: corrected

**Review by tpowell-progress — APPROVED** [review]

**Pull request merged**